docs/source/algorithms/IntegratePeaksShoeboxTOF-v1.rst

```diff
@@ -0,0 +1,86 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This is an algorithm to integrate single-crystal Bragg peaks in a :ref:`MatrixWorkspace <MatrixWorkspace>` by
+summing up counts in a shoebox of size ``NRows`` and ``NCols`` on the detector and ``NBins`` along the Time-of-flight
+(TOF) direction which can be specified in one of two ways:
+
+1. Provide ``NBins`` - number of TOF bins in the kernel
+
+2. Setting ``GetNBinsFromBackToBackParams=True`` and providing ``NFWHM`` - in which case ``NBins``  will be NFWHM x FWHM
+   of a :ref:`BackToBackExponential <func-BackToBackExponential>` peak at the center of each detector panel/bank at the
+   middle of the spectrum.
+
+Note to use method 2, back-to-back exponential coefficients must be defined in the Parameters.xml file for the
+instrument.
+
+The integration requires a background shell with negative weights, there are approximately the same number of bins in
+the background shell as in the peak region.
+
+The algorithm proceeds as follows:
+
+1. Take a window of the data, ``NShoeboxInWindow`` the size of the shoebox (``NRows`` x ``NCols`` x ``NBins``)
+
+2. Convolve the shoebox kernel with the data in the window to find the position with largest Intensity/sigma
+   (closest to the predicted peak position than to any other peaks in the table).
+
+3. Integrate using the shoebox at the optimal position
+
+4. If the peak is strong (Intensity/sigma < ``WeakPeakThreshold``) and ``OptimiseShoebox=True`` then optimise the shoebox
```

Comment:
Here do you mean `Intensity/sigma > WeakPeakThreshold`?

Reply:
I mean the ratio of the integrated intensity and sigma (the error in the peak intensity) is greater than the input parameter `WeakPeakThreshold`. The term `I/sigma`  and `Intensity/sigma` are very common terms in crystallography.

Reply:
Sorry, I think I didn't explain myself very well. What I mean is that in the document there is a typo in the direction of the operator , it should be '>' instead of '<'.

Reply:
haha, yes good point! Sorry i'll fix!